packages/core/src/components/SingleSelect/useKeyboardNavigation.tsx

```diff
@@ -74,6 +77,13 @@ export const useKeyboardNavigation = (props: Props) => {
             hoverOption(nestedOptions.current.pop()!);
     }, [leftPress, areOptionsVisible]);
 
+    useEffect(() => {
```

Comment:
Could you please write tests for this also ?

Reply:
Added